cookieapp.js

```diff
@@ -0,0 +1,200 @@
+'use strict'
+
+//First store object
+var pikeStore = {
+
+  storeName: '1st and Pike',
+  storeHours: ['6am', '7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm'],
```

Comment:
If store hours are the same for all stores, consider storing them as a global.  If store hours were different between locations it makes sense to store them as a property (or properties - open/close) on the object.